src/ring_theory/norm.lean

```diff
@@ -127,7 +127,7 @@ end eq_prod_roots
 
 section eq_zero_iff
 
-lemma norm_eq_zero_iff_of_basis [is_domain S] (b : basis ι R S) {x : S} :
+lemma norm_eq_zero_iff_of_basis [is_domain R] [is_domain S] (b : basis ι R S) {x : S} :
```

Comment:
Why do you end up needing to add `is_domain R` here?

Reply:
Because it's no longer in the `variables` statement at the top of the file, and this proof seems to still require it.

Reply:
Oh, of course.